SwiftCompilerSources/CMakeLists.txt

```diff
@@ -171,7 +171,7 @@ function(add_swift_compiler_modules_library name)
   endforeach()
 
   # Create a static library containing all module object files.
-  add_library(${name} STATIC ${all_obj_files})
+  add_library(${name} STATIC force_lib.c ${all_obj_files})
```

Comment:
Can you add this conditionally only if built with --xcode?
This would also make it clear in which cases we need that.

Also, can you add a comment the workaround here?

Reply:
I've bracketed the fix in an if(XCODE) and borrowed a comment from what appears to be a previous attempt at fixing this problem further down the file. Should I remove that fix also? (seems to work without it)
```
  # Xcode does not compile libraries that contain only object files.
  # Therefore, it fails to create the static library. As a workaround,
  # we add a dummy script phase to the target.
  if (XCODE)
    add_custom_command(TARGET ${name} POST_BUILD 
      COMMAND ""
      COMMENT "Dummy script phase to force building this target")
  endif()